.github/workflows/dialyzer-cache.yml

```
@@ -0,0 +1,78 @@
+      id: plt-cache
+      with:
+        path: priv/plts
+        key: ${{ runner.os }}-plt-${{ env.ELIXIR_VERSION }}-${{ env.OTP_VERSION }}-${{ hashFiles('**/mix.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-plt-${{ env.ELIXIR_VERSION }}-${{ env.OTP_VERSION }}-
+    
+    - name: Install system dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y build-essential libbz2-dev bc erlang-dev
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)